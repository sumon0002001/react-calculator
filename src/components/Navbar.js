import React from 'react';
import { Link } from 'react-router-dom';
const NavBar = () => (
    <header>
      <h1>Math Magicians</h1>
      <nav>
        <ul>
          <li>
            <Link to="/" className="nav-link">Home</Link>
          </li>
          <li>
            <Link to="/calculator" className="nav-link">Calculator</Link>
          </li>
          <li>
            <Link to="/quete" className="nav-link">Quete</Link>
          </li>
        </ul>
      </nav>
    </header>
  );
export default NavBar;
