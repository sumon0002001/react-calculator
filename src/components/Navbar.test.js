import React from 'react';

import { BrowserRouter } from 'react-router-dom';
import '@testing-library/jest-dom';
import { render, screen } from '@testing-library/react';
import Navbar from './Navbar';

const content = render(<BrowserRouter><Navbar /></BrowserRouter>);

test('Renders the Navbar page', () => {
  expect(content.container).toMatchSnapshot();
  expect(screen.getByText('Calculator')).toBeInTheDocument();
});
