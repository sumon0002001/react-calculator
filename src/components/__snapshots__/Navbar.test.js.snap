// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders the Navbar page 1`] = `
<div>
  <header>
    <h1>
      Math Magicians
    </h1>
    <nav>
      <ul>
        <li>
          <a
            class="nav-link"
            href="/"
          >
            Home
          </a>
        </li>
        <li>
          <a
            class="nav-link"
            href="/calculator"
          >
            Calculator
          </a>
        </li>
        <li>
          <a
            class="nav-link"
            href="/quete"
          >
            Quete
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;
